### DISCUSSION TITLE
Bugfixes and Cleanup

### DISCUSSION THREAD

**Discussion by hyfabi** [HyFabi/Blocket#57] [Ideas]
# Changelog #56 
- Fix lagspikes while running around https://github.com/blocketGame/Blocket/pull/54 
- Clean code https://github.com/blocketGame/Blocket/pull/55 
- Add In-Game-Commands    - `/gm {-1...0}` (Gamemode) #52 
   - `/give {itemId}`
   - `/collision {true/false}`
- Rewrote movement due to crapy one before https://github.com/blocketGame/Blocket/pull/52 
- Implemented "Moving Item" (the item that the player is holding) around player https://github.com/blocketGame/Blocket/pull/53

If you have any issues by profiles: Try restart your game!

Your welcome to write bug-issues! :) :trollface: 

<hr /><em>This discussion was created from the release <a href='https://github.com/blocketGame/Blocket/releases/tag/v1.6-alpha'>Bugfixes and Cleanup</a>.</em>

**Comment by Jojotschl** [comment]
Das Spiel ist quasi fertig für den Release!!!

**Reply by PHCaesar** [reply]
*Confused Screaming* es fehlt noch viel zu viel????
Es sind aber auch einige dinge im build verbugged die wir eigentlich bereits implementiert haben

**Reply by Jojotschl** [reply]
hmmmmm

**Reply by hyfabi** [reply]
Natürlich
